app/views/shared/_downtime_warning.html.haml

```diff
@@ -1,10 +1,9 @@
 - if display_downtime_warning?
-  .moj-banner
-    %span.visuallyhidden
-      = t('common.warning')
-    %svg.moj-banner__icon{fill: "currentColor", role: "presentation", focusable: "false", xmlns: "http://www.w3.org/2000/svg", viewBox: "0 0 25 25", height: "25", width: "25"}
-      %path{ d: "M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8 C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z"}
-
-    .moj-banner__message
-      %h2.govuk-heading-m
+  .govuk-notification-banner{"aria-labelledby" => "govuk-notification-banner-title", "data-module" => "govuk-notification-banner", :role => "region"}
```

Comment:
Guide: could we user the new hash syntax, for example `role: 'region'` instead of the arrow hash syntax `:role => "region"`